i18n/en.json

```diff
@@ -13,5 +13,6 @@
 	"wikibasereconcileedit-editendpoint-unspecified-entity-input-version": "entity input version must be specified",
 	"wikibasereconcileedit-editendpoint-invalid-entity-input-version": "unknown entity input version",
 	"wikibasereconcileedit-editendpoint-qualifiers-references-not-supported": "Qualifiers and References are not currently supported",
-	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Entity must have at least one statement for the reconciliation Property"
+	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Entity must have at least one statement for the reconciliation Property",
+	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must be of type value"
```

Comment:
“must be of type value” sounds pretty technical to me; maybe “must contain a value” would be clearer?

Reply:
Done.